python/cudf/cudf/tests/test_rolling.py

```diff
@@ -220,7 +215,7 @@ def test_rolling_var_std_large(agg, ddof, center, seed, window_size):
 
 
 @pytest.mark.xfail(
-    condition=not PANDAS_GE_130,
+    condition=not PANDAS_LT_140,
```

Comment:
This is changing the version _and_ the direction of the bounds. Is this correct? Before: `not >=1.3.0` (so `< 1.3.0`). After: `not <1.4.0` (so `>=1.4.0`). Does this break compatibility with pandas 1.2, 1.3, ...?

More broadly, do we have a commitment to support all pandas versions 1.0+ in our tests? I think we should re-evaluate our pandas pinning strategy in that light and attempt to improve test coverage in that direction. It should be much more straightforward to test multiple dependency versions after the switch to GitHub Actions (#12002).

Reply:
It was actually fixed in 1.3.0 and broken in 1.4.0 until 1.5.0, so the condition has to be:
```
condition=not PANDAS_GE_130 or (PANDAS_GE_140 and not PANDAS_GE_150),
```

Reply:
@bdice is also right, It might be worth revisiting our back-word compatible pytest execution once github actions are enabled. But yeah not in scope of this PR.

Reply:
Hmm, given these statements and also finding more issue I think I should just move to 1.5.1 and close this PR.   What do you all think ?

Reply:
If you are in a dev environment, yes upgrading to 1.5.1 would be ideal. We can only guarantee 1.4 behavior for when our max support was 1.4 i.e., Jan 2022.

Reply:
Ok.  Closing the PR.  Sorry for the noise!